From 02f7cbb9a6deaa24f106ff6e3d38834ecd5003bc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Tue, 24 May 2016 11:29:05 +0200
Subject: [PATCH] vtpmmgr: fix misleading indentation.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Organization: Invisible Things Lab
Cc: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>

gcc-6 adds -Wmisleading-indentation which found these issues.

log.c: In function 'tpm_get_error_name':
log.c:146:3: error: this 'for' clause does not guard... [-Werror=misleading-indentation]
   for (i = 0; i < sizeof(error_msgs)/sizeof(error_msgs[0]); i++)
   ^~~
log.c:150:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'for'
     return("Unknown Error Code");
     ^~~~~~

disk_read.c: In function 'parse_root_key':
disk_read.c:128:5: error: this 'else' clause does not guard... [-Werror=misleading-indentation]
     else
     ^~~~
disk_read.c:131:2: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'else'
  if (rc)
  ^~

Signed-off-by: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>
---
 stubdom/vtpmmgr/disk_read.c | 24 ++++++++++++------------
 stubdom/vtpmmgr/log.c       |  2 +-
 2 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/stubdom/vtpmmgr/disk_read.c b/stubdom/vtpmmgr/disk_read.c
index 944d3ff..6c58685 100644
--- a/stubdom/vtpmmgr/disk_read.c
+++ b/stubdom/vtpmmgr/disk_read.c
@@ -67,7 +67,7 @@ static int find_group_key(struct mem_group *dst,
 		const struct mem_tpm_mgr *parent)
 {
 	int i, rc, rv = 1;
-    unsigned int olen;
+	unsigned int olen;
 	struct hash160 buf;
 	struct disk_group_sealed_data sealed;
 
@@ -90,11 +90,11 @@ static int find_group_key(struct mem_group *dst,
 		if (memcmp(&buf, &cfg->digest_release, 20))
 			continue;
 
-        /*TPM 2.0 unbind | TPM 1.x unseal*/
-        if (hw_is_tpm2())
-            rc = TPM2_disk_unbind(&sealed, &olen, cfg);
-        else
-            rc = TPM_disk_unseal(&sealed, sizeof(sealed), cfg);
+		/*TPM 2.0 unbind | TPM 1.x unseal*/
+		if (hw_is_tpm2())
+			rc = TPM2_disk_unbind(&sealed, &olen, cfg);
+		else
+			rc = TPM_disk_unseal(&sealed, sizeof(sealed), cfg);
 
 		if (rc)
 			continue;
@@ -119,14 +119,14 @@ static int find_group_key(struct mem_group *dst,
 static int parse_root_key(struct mem_tpm_mgr *dst, struct disk_seal_entry *src)
 {
 	int rc;
-    unsigned int olen;
+	unsigned int olen;
 	struct disk_root_sealed_data sealed;
 
-    /*TPM 2.0 unbind | TPM 1.x unseal*/
-    if (hw_is_tpm2())
-        rc = TPM2_disk_unbind(&sealed, &olen, src);
-    else
-        rc = TPM_disk_unseal(&sealed, sizeof(sealed), src);
+	/*TPM 2.0 unbind | TPM 1.x unseal*/
+	if (hw_is_tpm2())
+		rc = TPM2_disk_unbind(&sealed, &olen, src);
+	else
+		rc = TPM_disk_unseal(&sealed, sizeof(sealed), src);
 
 	if (rc)
 		return rc;
diff --git a/stubdom/vtpmmgr/log.c b/stubdom/vtpmmgr/log.c
index a82c913..c1bc8f3 100644
--- a/stubdom/vtpmmgr/log.c
+++ b/stubdom/vtpmmgr/log.c
@@ -147,5 +147,5 @@ const char* tpm_get_error_name (TPM_RESULT code) {
     if (code == error_msgs[i].code)
       return error_msgs[i].code_name;
 
-    return("Unknown Error Code");
+  return("Unknown Error Code");
 }
-- 
2.1.0

